server/src/email/templates/noncommercial/default-application-details.es6

```diff
@@ -7,7 +7,8 @@ module.exports = {
     return `
       Application details
       *********************************
-
+      
+      app status: ${application.status}
```

Comment:
For content consistency, should this be `Application status: ${application.status}`?

Reply:
oh, this was a debugging statement- will remove!

Reply:
but actually it seems worthwhile to have ha!

Reply:
realizing this is covered by the conditional in the link text to the app